src/app/components/observation/registration-view-config.ts

```diff
@@ -0,0 +1,128 @@
+import { RegistrationTid } from 'src/app/modules/common-registration/registration.models';
+import { RegistrationViewConfig } from './registration-view-config.model';
+import { SummaryComponent } from './summary/summary.component';
+import { getSummaryInputs } from './summary/get-summary-input';
+import { isObservationModelEmptyForRegistrationTid } from 'src/app/modules/common-registration/registration.helpers';
+import { AvalancheEvaluationViewComponent } from './registrations/avalanche-evaluation-view/avalanche-evaluation-view.component';
+import { isEmpty } from 'src/app/modules/common-core/helpers';
+import { AvalancheProblemViewComponent } from './registrations/avalanche-problem-view/avalanche-problem-view.component';
+
+/**
+ * Konfig for hvilke komponenter som skal vises for skjemaer i en registrering.
+ *
+ * Konfigurasjonen er veldig eksplisitt satt opp.
+ * Et alternativ hadde vært å feks falle tilbake på en Summary-baserte visning hvis
```

Comment:
Ja, den er eksplisitt, og det gjør den kanskje litt sårbar for endringer i skjema. 
Hva om vi enten har fallback på Summary-basert visning eller gjør en validering av at vi har med alle skjema i RegistrationTid. Om det dukker opp et nytt skjema i RegistrationTid som vi ikke har i REGISTRATION_VIEW_CONFIG, kunne vi vist navn på skjema og deretter en feilmelding eller noe?

Reply:
Tror jeg dropper en sånn løsning for nå, tenker det er så sjeldent vi legger til nye skjema at det blir unødvendig. Hvis vi plutselig begynner å legge til masse skjema kan vi revurdere.

Reply:
Tror jeg dropper en sånn løsning for nå, tenker det er så sjeldent vi legger til nye skjema at det blir unødvendig. Hvis vi plutselig begynner å legge til masse skjema kan vi revurdere.